includes/data-stores/abstract-wc-order-data-store-cpt.php

```diff
@@ -222,6 +222,11 @@ protected function update_post_meta( &$order, $force = false ) {
 			'_order_version'      => 'version',
 			'_prices_include_tax' => 'prices_include_tax',
 		);
+
+		if ( false === $force ) {
+			$force = (bool) get_post_meta( $order->get_id(), '_woocommerce_force_meta_update', true );
+		}
+
```

Comment:
How about moving this upstream? Since there's a `$force` method arg, wouldn't it be cleaner to call the method with the correct `$force` value?